api/utils/taskmanager.js

```diff
@@ -228,6 +228,18 @@ var request = require("request");
         options.db.collection("long_tasks").findOne({_id:options.id}, callback);
     };
     
+    /**
+    * Edit specific task
+    * @param {object} options - options for the task
+    * @param {object} options.db - database connection
+    * @param {object} options.task_id - ID of the target task 
+    * @param {string} options.data - data of the task want to modify
+    */
+    taskmanager.editTask = function(options, callback){
+        options.db = options.db || common.db;
+        options.db.collection("long_tasks").update({_id: options.task_id}, {$set: options.data}, callback);
+    };
+    
```

Comment:
Let's keep parameter naming consistent with other methods, like `saveResult`, so not task_id, but id, etc